pkg/nvml/nvml.go

```diff
@@ -102,6 +102,25 @@ func nvmlSystemGetProcessName(Pid uint32, Name *byte, Length uint32) Return {
 	return __v
 }
 
+// nvmlSystemGetHicVersion function as declared in nvml/nvml.h
+func nvmlSystemGetHicVersion(HwbcCount *uint32, HwbcEntries *HwbcEntry) Return {
```

Comment:
I don't see a corresponding top-level or interface implementation for this.

Reply:
Its there already, actually:
https://github.com/NVIDIA/go-nvml/blob/main/pkg/nvml/system.go#L53

In each of these versions bumps a whole bunch of functions were moved around, so there are a bunch of `+`s and `-`s on functions that are really just moves within the file.